index.js

```diff
@@ -49,11 +33,13 @@ class Analytics {
     this.queue = []
     this.writeKey = writeKey
     this.host = removeSlash(options.host || 'https://api.segment.io')
-    this.retryCount = options.retryCount || 3
     this.timeout = options.timeout || false
     this.flushAt = Math.max(options.flushAt, 1) || 20
     this.flushInterval = options.flushInterval || 10000
     this.flushed = false
+
+    // setup retries
```

Comment:
I think comment can be removed, since the code is pretty self-explanatory.

Reply:
Good point. Will remove.